htsget.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# Check if command exists
+if ! command -v python3 &>/dev/null; then
```

Comment:
Not needed if you are already checking for python3.6

Reply:
Added set -xe and also removed the requested code from the script.